statics/js/components/rule-detail.js

```diff
@@ -497,6 +497,16 @@ Vue.component('rule-detail', {
       }
     };
     this.handler = {
+      onPreInit: function(){},
```

Comment:
I don't think we need this as there is a check now:
https://github.com/skydive-project/skydive/blob/master/statics/js/components/topology.js#L399

Reply:
It wasn't my code, I just fixed this line because it caused crash of right panel...
Originally it had incorrect method signature.
I just didnt put it as a separate commit.
If i remove this line - there will be incorrect code...

Reply:
Since this PR it is useless no ?
https://github.com/skydive-project/skydive/pull/502

Reply:
Is it already merged?

If yes - I now will try to remove it.
Thanks

Reply:
Yes already